packages/wonder-blocks-modal/components/standard-modal.md

```diff
@@ -41,7 +40,7 @@ const styles = StyleSheet.create({
             title="Title"
             content={
                 <View style={styles.modalContent}>
-                    <Body tag="p">
```

Comment:
I removed these as paragraphs add vertical padding and made it confusing to reason about the visible spacing between things (like the header and the content).